bot.lua

```diff
@@ -4847,10 +4862,11 @@ commands["lua"] = {
 			ms = (os.clock() - ms) * 1000
 
 			if not success then
+				local user = message.member or message.author
```

Comment:
Instead of duplicating the code why don't you make this variable outside the chunks?